conf/proxysql.conf.example

```diff
@@ -118,8 +119,15 @@ mysql_query_rules =
     {
         rule_id=200
         active=1
-        match_pattern=".*read_only.*"
-        destination_hostgroup=10
+        match_pattern="SELECT @@global.read_only"
+        destination_hostgroup=30
+        apply=1
+    },
+    {
+        rule_id=200
```

Comment:
you should raise it, the rule id 200 already exist.

Reply:
The roule has been raised